prac_05/word_occurrences.py

```diff
@@ -0,0 +1,17 @@
+"""
+Word Occurrences
+Estimate: 10 minutes
+Actual: 22 minutes
+"""
+word_to_frequency = {}
+# text = input("Enter Text: ")
+text = "this is a collection of words of nice words this is a fun thing it is"
+words = text.split()
+for word in words:
+    frequency = word_to_frequency.get(word, 0)
+    word_to_frequency[word] = frequency + 1
```

Comment:
I could be wrong, but would it work if these were combined as one line, something like:
word_to_frequency[word] = word_to_frequency.get(word, 0) + 1

Reply:
Yep that works 😄